lighthouse-plugin-publisher-ads/tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "../node_modules/lighthouse/lighthouse-core/report/html/renderer/i18n.js",
     "../node_modules/lighthouse/lighthouse-core/lib/network-request.js",
     "../node_modules/lighthouse/lighthouse-core/lib/url-shim.js",
+    "../node_modules/lighthouse/lighthouse-core/util-commonjs.js",
```

Comment:
as discussed in chat, including `url-shim.js` in the line above this gives `tsc` a way into Lighthouse code instead of just treating it as a module. url-shim now imports `util-commonjs` and needs an explicit import of it as well to avoid `any` errors. Ideally none of these imports should be necessary but I assume it's done because LH doesn't publish any types yet.